bootstrap/bootstrap.sh

```diff
@@ -1,2 +1,35 @@
 #!/bin/sh
-echo "This is a placeholder for the TJBot boostrap script. Until it has been written, please follow the directions in README.md to configure your Raspberry Pi for TJBot."
+
+#----validation
+#if $EUID -ne 0; then
+#    echo "You need to install as root by using sudo";
+#    exit
+#fi
+
+#----force ipv4
+if ! grep -q "ipv6.disable=1" /boot/cmdline.txt; then
+    echo " ipv6.disable=1" | sudo tee -a /boot/cmdline.txt
+    echo "We just disabled ipv6 due better network compatibility. It will take effect after restart.";
+fi
+
+#----default DNS (google)
+if ! grep -q "nameserver 8.8.8.8" /etc/resolv.conf; then
+    echo "nameserver 8.8.8.8" | sudo tee -a /etc/resolv.conf
+    echo "nameserver 8.8.4.4" | sudo tee -a /etc/resolv.conf
+fi
+
+#----debian missing locale (just to remove warnigs)
+export LC_ALL="en_US.UTF-8"
+echo "en_US.UTF-8 UTF-8" | sudo tee -a /etc/locale.gen
+sudo locale-gen en_US.UTF-8
+
+#----update raspberry
+sudo apt-get update
+sudo apt-get -y dist-upgrade
+
+#----nodejs install
+curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
```

Comment:
I'll say we stick to setup_6.x ...
The last time i tried 7.x I had some installation errors. But 6.x works